pino.js

```diff
@@ -297,7 +288,31 @@ function pino (opts, stream) {
   var instance = Object.create(pinoPrototype)
   instance.stream = istream
   tools.defineLevelsProperty(instance)
-  tools.applyOptions.call(instance, iopts)
+
+  instance.stringify = iopts.stringify
```

Comment:
Moving these back here from `applyOptions` must not have the performance impact that originally necessitated them in `applyOptions`. That's probably because we aren't using `new` any longer :)